class_book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    rental = Rental.new(date, self, person)
+    @rentals.push(rental)
+    rental
```

Comment:
- [ ] Same for the book as pointed out :point_up:  in the previous comment. When a new Rental is created it adds itself to the book's rentals, thanks to this code `book.rentals << self` in the `Rental` class at line `8`. So pushing the new rental to the book's rentals is adding the same rental twice. Therefore I suggest you remove this code from the `add_rental` method.
  Example:
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

Reply:
- Remove unnecessary code from `add-rental` method to resolve bug.